### DISCUSSION TITLE
Support for other languages

### DISCUSSION THREAD

**Discussion by Waramoto** [AveYo/ShowMMR#12] [Ideas]
Hello! 
I noticed that the MMR showing doesn't work for some languages (for example Ukrainian). 
Can you add support for other languages in future versions please?

![image](https://github.com/AveYo/ShowMMR/assets/57632377/60b4ad3e-a8c3-4bb9-8973-48f8e9f28848)

**Comment by Waramoto** [comment]
Sorry, I didn't notice that this issue had already been raised in the "Issues" section.

**Reply by AveYo** [reply]
Have you tested with the last version?
I would appreciate a confirmation it's fixed.

**Reply by Waramoto** [reply]
Yes, it's fine now. Thank you!

![image](https://github.com/AveYo/ShowMMR/assets/57632377/61bd7ae1-2011-4167-aa32-c3d6278f3765)
